pages/mintKamon/index.tsx

```diff
@@ -1,33 +1,48 @@
 import type { NextPage } from 'next'
 import { useAccount } from 'wagmi'
-import { Button, Text, Heading } from '@chakra-ui/react'
+import { Button, Heading } from '@chakra-ui/react'
 import { useState } from 'react'
+import { useConnect } from 'wagmi'
+
 import { Layout } from '@/components/layouts/layout'
 import { GenerateImageForm } from '@/components/mintKamon/GenerateImageForm'
+import { useMounted } from '@/hooks/useMounted'
 
 const MintKamon: NextPage = () => {
-  const [{ data, error, loading }, disconnect] = useAccount()
+  const mounted = useMounted()
+  const { connect, connectors, error } = useConnect()
+  const [metaMask] = connectors
+  const { data } = useAccount()
+
   const [tokenURI, setTokenURI] = useState('')
   const [approved, setApprove] = useState(false)
 
-  if (!tokenURI) {
+  if (false && !tokenURI) {
```

Comment:
oops lol
but I will leave this one. without this line, we can get sense of connect wallet